src/routes/game/GameView.vue

```diff
@@ -842,7 +842,7 @@ export default {
      * @param player is the player object
      */
     kingCount(player) {
-      return player.faceCards.reduce((kingCount, card) => kingCount + (card.rank === 13 ? 1 : 0), 0);
+      return player?.faceCards?.reduce((kingCount, card) => kingCount + (card.rank === 13 ? 1 : 0), 0) ?? 0;
```

Comment:
This wasn't necessary for the scope, but it's been a common source of type errors for this computed to barf if `player` is undefined. Better to coalesce nicely to 0